src/features/MoviesBrowser/MovieDetails/Cast/index.js

```diff
@@ -10,13 +10,13 @@ import { selectMovieCreditsSlice } from "./creditsSlice";
 
 const Cast = () => {
   const movieCredits = useSelector(selectMovieCreditsSlice);
-
+console.log(movieCredits)
   return (
     <>
       <CastHeader>Cast</CastHeader>
```

Comment:
I changed Cast for Credits. Credits is overall component for cast and crew.

Reply:
There was also "Cast" in "main" when I downloaded it, so it copied the same to the branch. I hope there will be no major problems with merging.